sunspot/lib/sunspot/indexer.rb

```diff
@@ -9,8 +9,9 @@ module Sunspot
   #
   class Indexer #:nodoc:
 
-    def initialize(connection)
+    def initialize(connection, options = {})
       @connection = connection
+      @update_options = {params: options.delete_if { |key, value| value.nil? }}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.](https://www.codacy.com/app/joni-amir/sunspot/file/2482066248/issues/source?bid=3265104&fileBranchId=3265120#l14)